src/events.rs

```diff
@@ -60,7 +68,6 @@ pub enum Event {
     /// The parameter is true if app was suspended, and false if it has been resumed.
     Suspended(bool),
```

Comment:
Ah yes, the problem with `Suspended` and `Awakened` is that I need to change the backends' code if I remove them.

As for the keyboard events, the windows compositor usually sends the event only to the currently focused window, so sending it to a specific window is how it's designed.